src/qt/forms/debugwindow.ui

```diff
@@ -1080,7 +1080,7 @@
               </property>
               <property name="minimumSize">
                <size>
-                <width>300</width>
+                <width>0</width>
```

Comment:
Hmm works too ,,, but a i kinda like short changes.
and b when edit with Qt designer the ui file, the default is to have at least a
`<property name="geometry">`
So for now i like to stay a bit conservative, since i can not test easy outside my cruft. 
so how about this?

``` diff
diff --git a/src/qt/forms/debugwindow.ui b/src/qt/forms/debugwindow.ui
index 8b7080083..1217ca3e2 100644
--- a/src/qt/forms/debugwindow.ui
+++ b/src/qt/forms/debugwindow.ui
@@ -1078,12 +1078,6 @@<property name="geometry">
                 <height>426</height>
                </rect>
               </property>
-              <property name="minimumSize">
-               <size><property name="geometry">
-                <width>300</width>
-                <height>0</height>
-               </size>
-              </property>
               <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_30">




I have written a shorty test script to test long flag lists, by spawn up 2 connected
 regtest nodes. 

``` bash
#!/bin/bash
params="-regtest -debug=1"
makeflags=-j4 
dir=/tmp/bitcoin-test
baseport=1234
testfix=0 #if 1 auto git pull the gui fix from Saibato PR #6 https://github.com/bitcoin-core/gui/commit/099defc58975e2ec0e8069b84cf852f7673122dd

if [ $testfix = 1 ]; then git pull --no-edit https://github.com/Saibato/gui fix-qt-disp-flags ;fi

t=$(echo "")
for i in {1..8} # number of flags to set active ( max 64 )
 do t+=$(echo "|(1<<$i)")
done

#Next change init.cpp to set many flags 
 
sed -i "s/ServiceFlags(NODE_NETWORK | NODE_NETWORK_LIMITED);/ServiceFlags(NODE_NETWORK | NODE_NETWORK_LIMITED $t);/g" ./src/init.cpp
#Now recompile the the code
make $makeflags

printf "######\n\n now wait for-torcontrol=127.0.0.1:9151 the nodes to spun up ...\ntest with window>Network Traffic>Peers\n######"

# now revert the set many node flags in init.cpp 
sed -i "s/ServiceFlags(NODE_NETWORK | NODE_NETWORK_LIMITED $t)/ServiceFlags(NODE_NETWORK | NODE_NETWORK_LIMITED)/g" ./src/init.cpp
if [ $testfix = 1 ]; then git reset --hard HEAD~1 ;fi

for i in 1 2
do port2=$((3-$i))
 rm -rf $dir$i && mkdir $dir$i
 ./src/qt/bitcoin-qt $params  -bind=127.0.0.1 $(if [ $i = 1 ]; then echo -server;fi) -wallet=test -listen -port=$(echo $baseport$i) -datadir=$dir$i -connect=127.0.0.1:$(echo $baseport$port2) &
done